magenta-lib/src/main/scala/magenta/input/DeploymentResolver.scala

```diff
@@ -0,0 +1,78 @@
+package magenta.input
+
+
+object DeploymentResolver {
+  val DEFAULT_REGIONS = List("eu-west-1")
+
+  def resolve(config: RiffRaffDeployConfig): List[Either[(String, String), Deployment]] = {
```

Comment:
`Either` is going to be right-biased in 2.12, so cats' `Xor` is going away.

I really don't know what those `String`s are, so would definitely value a case class. 

If in doubt, stick a type on it.